introduction.md

```diff
@@ -1,3 +1,6 @@
 This is an introduction.
 
 My favorite path is Ruby.
+
+Deadly skills:
+* Everything
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?